bpf/lib/nat.h

```diff
@@ -1698,26 +1701,26 @@ snat_v6_prepare_state(struct __ctx_buff *ctx, struct ipv6_nat_target *target)
 	struct ipv6hdr *ip6;
 
 	if (!revalidate_data(ctx, &data, &data_end, &ip6))
-		return false;
+		return NAT_PUNT_TO_STACK;
```

Comment:
```suggestion
		return DROP_INVALID;
```

Reply:
Right, I paid attention to it for IPv4 but missed this one. Good catch